_pages/business-units/18f/projects-partners/collaboration-tools.md

```diff
@@ -0,0 +1,168 @@
+* **Skype**
+* **BlueJeans**
+</details>
+
+### Sharing Files
+
+**[Google Drive]({{site.baseurl}}/google-drive/)** is a storage and collaboration platform for spreadsheets, slide decks, and text documents. Documents on Google Drive are not public, but can be shared among team members. 18F can give your team access to our Google Drive so everyone can collaborate.
+
+<details>
+<summary>Compliance and records considerations</summary>
+* [Google G Suite has a FedRAMP Agency ATO](https://marketplace.fedramp.gov/#/product/google-g-suite) Moderate Impact authorized since January 2016. Agencies using the service include GSA, DOE, DHS, and others.
+</details>
+
+Here are some common barriers to using Drive and some workarounds to try:
+* **IF** the partner doesn’t have a government Google account, **THEN** create GACA accounts for partners, or share files via services below or via email.
+* **IF** they’re blocked by firewall/security issues, **THEN** try options below.
+
+<details>
+<summary>Other file sharing services to consider or ask your partner about</summary>
+* **Microsoft SharePoint:** a browser-based collaboration and documentation management system, similar to Google Drive. Recent releases of SharePoint provide allow teams to rapidly create internal websites (like Google Sites) and collaboratively create and manage content in documents, spreadsheets, and slides.  
+* **[Max.gov](https://portal.max.gov/portal/home):** a government-wide suite of advanced collaboration, information sharing, data collection, publishing, business intelligence, and authentication tools and services used to facilitate cross-government collaboration and knowledge management.
+* **[DOD SAFE](https://safe.apps.mil)**, (Secure Access File Exchange): SAFE is a service to exchange unclassified files that are too large to share via email.  SAFE can handle files up to 8.0 GB.
+* **[Intelink’s](https://www.intelink.gov/my.policy):** IntelDocs, allows users to share unclassified files. The service has a file size limit of 100MB and folder limit of 1GB.  
+</details>
+
+### Tracking project tasks
+
+**[Trello]({{site.baseurl}}/trello/)** is a project management tool that organizes information into lists. Teams often use it to track specific tasks through different stages (such as backlog, in progress, in review, and done). Trello boards can be private or public.
+
+<details>
+<summary>Compliance and records considerations:</summary>
+* GSA Tech Portfolio is working on a FedRAMP application as of January 2020. Members of TTS can use it and will be notified if anything changes.
+* Trello boards should be considered records.
+</details>
+
+<details>
+<summary>Trello alternatives</summary>
+* **Zenhub:** Not recommended from a compliance perspective. No FedRAMP in progress, no GSA IT approval for use of the service. From a user perspective, it’s great that it works within GitHub and provides higher-level scoping and linking (e.g., epics and dependencies) than Jira does.
```

Comment:
Minor, but Jira absolutely provides mechanisms for linking, dependencies and creating Epics.